docs/threat_models/treasury/treasury.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+header Agoric Operating Systems
+footer v0.0.1
+title Treasury
+
+skinparam componentStyle rectangle
+skinparam defaultTextAlignment center
+
+
+actor "TreasuryCreator aka Governance" as tc
+
+actor Borrower
+
+node AdjustBalancesInvitation
+node "CloseVaultInvitation"
+node LiquidationPromise
+node LiquidationSeat {
+    circle getPayoff
+    circle getPayoffs
+}
+node Treasury {
+    node AMM
+    node "publicAPI" {
+        circle getCollaterals
+        circle makeLoanInvitation
+        circle getRunIssuer
+        circle getAMM
+        getAMM -d-> AMM
+    }
+    node "creatorFacet" {
+        circle makeAddTypeInvitation
+    }
+}
+node "Vat" {
```

Comment:
`Vat` wrapping around Vault but not Treasury needs to be changed. All of the vaults are in the same vat as the treasury